homeassistant/components/apple_tv/remote.py

```diff
@@ -44,22 +49,19 @@ async def async_turn_off(self, **kwargs):
 
     async def async_send_command(self, command, **kwargs):
         """Send a command to one device."""
+        num_repeats = kwargs[ATTR_NUM_REPEATS]
+        delay = kwargs.get(ATTR_DELAY_SECS, DEFAULT_DELAY_SECS)
+
         if not self.is_on:
             _LOGGER.error("Unable to send commands, not connected to %s", self._name)
             return
 
-        for single_command in command:
-            _LOGGER.info("Sending command %s", single_command)
-            if single_command.startswith("delay="):
-                delay = single_command[6:]
-                try:
-                    float(delay)
-                except OverflowError:
-                    _LOGGER.error("Command delay must be a numeric %s", single_command)
+        for _ in range(num_repeats):
+            for single_command in command:
+                _LOGGER.info("Sending command %s", single_command)
+                if not hasattr(self.atv.remote_control, single_command):
```

Comment:
Use `getattr` with a default `None` as third argument and check the return value before calling it instead.

Reply:
@MartinHjelmare Thanks. Modified to use `getattr` as suggested.